src/components/calcite-shell-panel/calcite-shell-panel.tsx

```diff
@@ -24,6 +26,16 @@ export class CalciteShellPanel {
    */
   @Prop({ reflect: true }) collapsed = false;
 
+  /**
+   * This property makes the content area appear like a "floating" panel.
+   */
+  @Prop({ reflect: true }) detached = false;
+
+  /**
+   * This sets limits the height of the content area. It only applies when detached is true.
+   */
+  @Prop({ reflect: false }) detachedScale: "s" | "m" | "l" = "m";
+
   @Watch("collapsed")
```

Comment:
Nitpick: This watcher should be moved right bellow its associated prop, no?

Reply:
yea, ideally.